source/org/zfin/uniprot/handlers/ReportLostUniProtsHandler.java

```diff
@@ -132,8 +135,9 @@ public void handle(Map<String, RichSequenceAdapter> uniProtRecords, Set<UniProtL
      *
      * @param lostUniProt
      * @param action
+     * @param context
      */
-    private void setActionTitleAndDetailsForGenPeptGenBank(DBLinkSlimDTO lostUniProt, UniProtLoadAction action) {
+    private void setActionTitleAndDetailsForGenPeptGenBank(DBLinkSlimDTO lostUniProt, UniProtLoadAction action, UniProtLoadContext context) {
         Map<String, String> genBankMap = new HashMap<>();
         genBankMap.put("A0PGL6","ZDB-GENE-060818-12");
```

Comment:
Why are those two genes hard-coded?

Reply:
This is meant to capture some of the uniprot/gene mappings that were made by the legacy load. It is meant to be temporary since over time we will either decide to keep them (using manual curation), or remove them manually.